README.md

```diff
@@ -1452,7 +1452,21 @@ var event = await query.executeQuery();
 console.log(`>> Event: ${JSON.stringify(event)}`);
 ```
 
-
+# File Upload
```

Comment:
As discussed previously, can you indicate the limitation, i.e. in which conditions the API works (for instance, that it is only supported when used in the browser)